openfisca_web_api_preview/app.py

```diff
@@ -128,7 +131,7 @@ def apply_headers(response):
     @app.after_request
     def track_requests(response):
         if tracker:
-            tracker.track(request.url)
+            tracker.track(request.url, request.remote_addr)
```

Comment:
The IP address should not be sent if there is a ["Do not track" header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT)